packages/kbn-ai-playground/components/open_ai_key_callout.tsx

```diff
@@ -0,0 +1,49 @@
+import { EuiButton, EuiCallOut } from "@elastic/eui";
+import { css } from "@emotion/react";
+import { i18n } from "@kbn/i18n";
+
+import React, { Dispatch, SetStateAction } from "react";
+
+interface ListProps {
+  setIsOpenAIFlyOutOpen: Dispatch<SetStateAction<boolean>>;
+}
+
+export const OpenAIKeyCallout: React.FC<ListProps> = ({ setIsOpenAIFlyOutOpen }: ListProps) => {
+  return (
+    <EuiCallOut
+      css={css`
+        width: 55%
```

Comment:
I suppose it should not be here
All offsets already settled in a parent block
We can check it together